breathecode/authenticate/tests/urls/tests_me_invite.py

```diff
@@ -0,0 +1,164 @@
+        response = self.client.put(url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.json(), [
+            {
+                'created_at': self.datetime_to_iso(model1['user_invite'].created_at),
+                'email': 'a@a.com',
+                'first_name': None,
+                'id': 1,
+                'invite_url': f"http://localhost:8000/v1/auth/member/invite/{model1['user_invite'].token}",
+                'last_name': None,
+                'sent_at': None,
+                'status': 'ACCEPTED',
+                'token': model1['user_invite'].token
+            },
+            {'created_at': self.datetime_to_iso(model2['user_invite'].created_at),
+             'email': 'a@a.com',
+             'first_name': None,
+             'id': 2,
+             'invite_url': f"http://localhost:8000/v1/auth/member/invite/{model2['user_invite'].token}",
+             'last_name': None,
+             'sent_at': None,
+             'status': 'ACCEPTED',
+             'token': model2['user_invite'].token
+             }])
+        self.assertEqual(self.all_user_invite_dict(), [
+            {
+                'academy_id': 1,
+                'author_id': 1,
+                'cohort_id': 1,
+                'email': 'a@a.com',
+                'first_name': None,
+                'id': 1,
+                'last_name': None,
+                'phone': '',
+                'role_id': 'potato',
+                'sent_at': None,
+                'status': 'ACCEPTED',
+                'token': model1['user_invite'].token
+            },
+            {
+                'academy_id': 1,
+                'author_id': 1,
+                'cohort_id': 2,
+                'email': 'a@a.com',
+                'first_name': None,
+                'id': 2,
+                'last_name': None,
+                'phone': '',
+                'role_id': 'potato',
+                'sent_at': None,
+                'status': 'ACCEPTED',
+                'token': model2['user_invite'].token
+            }])
+
+    def test_invite_change_status_to_accepted_invitations_not_matched(self):
+        """Test academy/user/me/invite"""
+        self.headers(academy=1)
+
+        base = self.generate_models(
+            academy=True, capability='crud_invite', authenticate=True, role='potato', user_kwards={'email': 'a@a.com'})
+
+        invite_kwargs = {
+            'status': "ACCEPTED",
+            'email': 'a@a.com'
+        }
+
+        model1 = self.generate_models(authenticate=True, profile_academy=True, user_invite=True,
+                                      user_invite_kwargs=invite_kwargs, models=base)
+
+        model2 = self.generate_models(authenticate=True, profile_academy=True, user_invite=True,
+                                      user_invite_kwargs=invite_kwargs, models=base)
+
+        url = reverse_lazy('authenticate:user_me_invite') + '?id=1,2'
+        response = self.client.put(url)
```

Comment:
We need check the response should be equal to `[]`

Reply:
![Screen Shot 2021-06-22 at 4 55 48 PM](https://user-images.githubusercontent.com/60261389/122998279-d81a3880-d37a-11eb-9d79-946a66f87feb.png)